api/environments/models.py

```diff
@@ -69,6 +69,10 @@ class Environment(LifecycleModel):
     allow_client_traits = models.BooleanField(
         default=True, help_text="Allows clients using the client API key to set traits."
     )
+    updated_at = models.DateTimeField(
+        default=timezone.now,
+        help_text="It is a proxy for changes that affect a value of a given flag in the environment",
```

Comment:
Hmm... I'm not sure that's exactly correct, is it? I would say something more like: "Shows when the environment was last updated. Also tracks changes to related entities, e.g. FeatureStates."